src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/LockWheels.java

```diff
@@ -0,0 +1,26 @@
+package edu.greenblitz.tobyDetermined.commands.swerve;
+
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.greenblitz.utils.PigeonGyro;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+// import edu.wpi.first.math.geometry.Rotation2d;
+public class LockWheels extends SwerveCommand{
+	
+	PigeonGyro pigeon;
+
+
+	public LockWheels(){}
+
+	@Override
+	public void initialize(){
+		pigeon = swerve.getPigeonGyro();
+		new RotateAllWheelsToAngle(Math.PI / 2 - pigeon.getYaw()).schedule();
```

Comment:
Not crucial but better to use math deg to radians

Reply:
done!